sale_advance_payment/tests/test_sale_advance_payment.py

```diff
@@ -117,7 +118,7 @@ def setUpClass(cls):
             }
         )
 
-    def test_sale_advance_payment(self):
+    def test_01_sale_advance_payment(self):
```

Comment:
FYI numbering function does not ensure execution order.

Reply:
I thought it did, the test aren't executed in alphabetical order?